courses/beginner/10.md

```diff
@@ -48,207 +41,183 @@ control 'nginx-modules' do
   impact 1.0
-  title 'NGINX configuration'
-  desc 'The NGINX config file should owned by root, be writable only by owner, and not writeable or and readable by others.'
+  title 'NGINX configuration file'
+  desc 'The NGINX config file should exist.'
 end
 
-include_controls 'my_nginx_overlay'
-```
+control 'nginx-conf-perms' do
+  impact 1.0
+  title 'NGINX configuration permissions'
+  desc 'The NGINX config file should owned by root, be writable only by owner, and not writeable or and readable by others.'
+end
 
-```ruby
-# my_nginx_overlay
-control 'first-overlay-control'
-control 'second-overlay-control'
+control 'nginx-shell-access' do
+  impact 1.0
+  title 'NGINX shell access'
+  desc 'The NGINX shell access should be restricted to admin users.'
 ```
 
-In the example above, every time `my_nginx` profile is executed, all the controls from `my_nginx_overlay` are also executed. Therefore, the following controls would be executed:
+### Defining the Profile Dependency  
 
- - nginx-version
- - nginx-modules
- - nginx-conf
- - first-overlay-control
- - second-overlay-control
+For a profile to use controls from another profile, the dependency needs to be included in the `depends` section of the `inspec.yml` file. For example, you can develop my_nginx_overlay that uses controls from my_nginx profile. In that case, the `depends` section of `inspec.yml` of my_nginx_overlay should list the name and location of my_nginx. One way of declaring the dependency is:
 
-### Skipping a Control From a Profile
+::: code-tabs
+@tab my_nginx_overlay
+```yaml
+depends:
+  - name: my_nginx
+    path: ../my_nginx
+    # {path relative to the overlay}
+```
+:::
 
-What if one of the controls from the included profile does not apply to your environment? Luckily, it is not necessary to maintain a slightly-modified copy of the included profile just to delete a control. The skip_control command tells InSpec to not run a particular control.
+Once defined in the `inspec.yml` file, controls from the included profiles can be used!
 
-```ruby
-# my_nginx profile
-control 'nginx-version' do
-  impact 1.0
-  title 'NGINX version'
-  desc 'The required version of NGINX should be installed.'
-end
+::: info Find out more
+[Profile Dependencies ](https://www.inspec.io/docs/reference/profiles/#sts=Profile%20Dependencies)
+:::
 
-control 'nginx-modules' do
-  impact 1.0
-  title 'NGINX modules'
-  desc 'The required NGINX modules should be installed.'
-end
 
-control 'nginx-conf' do
-  impact 1.0
-  title 'NGINX configuration'
-  desc 'The NGINX config file should owned by root, be writable only by owner, and not writeable or and readable by others.'
-end
+### Including All Controls
 
-include_controls 'my_nginx_overlay' do
-  skip_control 'second-overlay-control'
-end
-```
+With the `include_controls` in a profile, all controls from the named profile will be executed every time the including profile is executed. Below you can see an example of a profile and overlay.
```

Comment:
When I get to this section, it is unclear to me if we are starting a new way of doing the overlay compared to the `depends` section above, or if they work together, or so on. Can you add a sentence explaining why we are looking at `include_controls` or how it fits with what we just did above?

Reply:
Maybe you can make it most clear by adding the full inspec.yml file in the second tab of all of your code examples so we see that it has `depends` and `include_controls`

Reply:
Great suggestion! I've tweaked the language and added some context.